### DISCUSSION TITLE
Azure AD Graph deprecation

### DISCUSSION THREAD

**Discussion by NovemOps** [KelvinTegelaar/CIPP#873] [Q&A] [answered]
maybe I haven't looked at the right location for the answer, but with the pending deprecation of AzureADGraph (even though the date was just extended to December 2022) will the Directory.AccessAsUser.All, User.Read, and Directory.Read.All permissions be updated?  is Microsoft Graph the alternative?

**Comment by KelvinTegelaar** [comment, marked as answer]
No worries! We actually don't use that permission at all anymore, and you can remove it. The PowerShell script adds it because some of the PowerShell modules such as Teams, and MSOnline require it. CIPP itself is 100% Graph

**Reply by NovemOps** [reply]
Thanks Kevin. Is there a script to remove them since they're no longer needed or should I just search and remove them manually?  I setup cipp before there was a PowerShell script (assuming that's what you mean) which set the permissions, so this was all manually configured initially.
 Thanks in advance.
